ldi-core/ngsiv2-to-ld/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldi/services/NgsiV2ToLdTranslatorService.java

```diff
@@ -0,0 +1,48 @@
+package be.vlaanderen.informatievlaanderen.ldes.ldi.services;
+
+import be.vlaanderen.informatievlaanderen.ldes.ldi.exceptions.InvalidNgsiLdContextException;
+import be.vlaanderen.informatievlaanderen.ldes.ldi.valuobjects.LinkedDataModel;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class NgsiV2ToLdTranslatorService {
+
+	private final String coreContext;
+	private final String ldContext;
+
+	public NgsiV2ToLdTranslatorService(String coreContext) {
+		this(coreContext, null);
+	}
+
+	public NgsiV2ToLdTranslatorService(String coreContext, String ldContext) {
+		this.coreContext = coreContext;
+		this.ldContext = ldContext;
+	}
+
+	public LinkedDataModel translate(String data) {
+		return translate(data, ldContext);
```

Comment:
This currently still returns the custom LinkedDataModel. I see you have a method to return a RDF model in its class, but this is used in a test :)